nautobot/dcim/migrations/0011_interface_status_data_migration.py

```diff
@@ -0,0 +1,61 @@
+# Generated by Django 3.1.14 on 2022-03-19 23:21
+
+from django.db import migrations, IntegrityError
+from nautobot.dcim.choices import InterfaceStatusChoices
+from nautobot.extras.management import export_statuses_from_choiceset
+
+
+def populate_interface_status(app, schema):
+
+    Status = app.get_model("extras.Status")
+    Interface = app.get_model("dcim.Interface")
+    ContentType = app.get_model("contenttypes.ContentType")
+
+    interface_content_type = ContentType.objects.get_for_model(Interface)
+    choices = export_statuses_from_choiceset(InterfaceStatusChoices)
+
+    # Create Interface Statuses and add dcim.Interface to its content_types
+    for choice_kwargs in choices:
+        try:
+            obj, created = Status.objects.get_or_create(**choice_kwargs)
+        except IntegrityError:
+            choice_kwargs.pop("color")
+            obj, created = Status.objects.get_or_create(**choice_kwargs)
```

Comment:
Why is this needed? Perhaps add an explanatory comment?

Reply:
Thinking of it, I don't think is needed